read-the-docs-500MM-pageviews.rst

```diff
@@ -0,0 +1,106 @@
+If you're really in a bind and it saves your a$$, then it's worth it's weight in gold.
+
+* 100,000,000 one dollar bills would get you to 6.79 miles, 
+or the approximate height that commercial airlines fly when transporting you from a to b[#f1].
+
+* Five times that height puts you halfway to the troposphere, or the lower atmosphere of planet Earth.  
+
+We're halfway to the atmosphere if you traded a page every page of documenation served by Read the Docs for a dollar. 
+
+* Don't like the money metaphor?  
+Paper is thicker than money.  
+A stack of 500 pages of old-school printed paper documentation that is outdated as soon as it's printed stands at 2 inches.  
+The current mountain of paper that Read the Docs climbed is *31.56 miles* tall.  
+
+You'd get pretty tired climbing that mountain.  
+Everest is only 5 miles tall compared to all of the paper that wasn't used articulating the size and scope of our milestone.  
+Run a 50K ultra-marathon vertically and you are just about there.
+
+Weird Math of Open Source
+=========================
+
+The weird math of open source is that unlike VC-funded and capitalized companies, 
+billionaires aren't made from these :doc:`herculean <read-the-docs-2015-stats>` accomplishments.  
+A lot of work is created, 
+and anyone who has operated enterprises with hundreds of millions of pageviews understand the dev and ops requirements, 
+which frequently includes scores of people and big budgets.
+
+Read the Docs has done this with a team of a core team of no more than two, 
+including Eric and Anthony, 
+the co-founders, 
+plus our brave and :doc:`valued contributors <https://github.com/rtfd/readthedocs.org/graphs/contributors>`
+who **bravely and boldly** continue to operate the service today.
+
+Today you can tip the math in favor of open source and more great documentation and better code.
+
+* Conscious individual?  Become a `Gold Member <https://readthedocs.org/accounts/gold/subscription/?>`, 
+you.
+Gold medalists sleep better at night.
+
+* Closed source, 
+private and public enterprises are invited to 
+participate in the Read the Docs revenue program, 
+which benefits the growth of Read the Docs to a billion pages of documentation viewed.  
+:doc:`Advertising on Read the Docs <state-of-the-docs-2015>`
+
+* Invest brain time instead of funny green paper from your wallet and read Roads and Bridges: :doc:`The Unseen Labor Behind Our Digital Infrastructure <http://readthedocs.org/sustainability/click/12/RGLMBScQnZcb/>` by Nadia Eghbal.  
+Society runs on open source software that is buckling under the demand.  
+
+* Invest `your time <http://docs.readthedocs.org/en/latest/contribute.html?>` 
+and get involved in open source by contributing to the project.
+
+
+Talk to us
+-----------
+
+If you have any feedback or opinions on any of this, continue to let 
+us know.  We're happy to have the feedback.
+
+.. _`Contact us`: hello@readthedocs.com
```

Comment:
or we could do a “contact me” with my scott@readthedocs.org on a blog post

i agree community building on the .org


> On Jul 27, 2016, at 10:14 AM, Eric Holscher <notifications@github.com> wrote:
> 
> In read-the-docs-500MM-pageviews.rst <https://github.com/rtfd/blog/pull/14#discussion_r72481480>:
> 
> > +:doc:`Advertising on Read the Docs <state-of-the-docs-2015>`
> > +
> > +* Invest brain time instead of funny green paper from your wallet and read Roads and Bridges: :doc:`The Unseen Labor Behind Our Digital Infrastructure <http://readthedocs.org/sustainability/click/12/RGLMBScQnZcb/>` by Nadia Eghbal.  
> > +Society runs on open source software that is buckling under the demand.  
> > +
> > +* Invest `your time <http://docs.readthedocs.org/en/latest/contribute.html?>` 
> > +and get involved in open source by contributing to the project.
> > +
> > +
> > +Talk to us
> > +-----------
> > +
> > +If you have any feedback or opinions on any of this, continue to let 
> > +us know.  We're happy to have the feedback.
> > +
> > +.. _`Contact us`: hello@readthedocs.com
> should probably use .org address here -- or figure out what makes sense to do in general.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/rtfd/blog/pull/14/files/dbccee6814c66fc3cf20c9d6ef100377f4995576#r72481480>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAmiVIKbxiQB42fDUijT8wsF6F90sfGnks5qZ5HwgaJpZM4JWWSG>.
>